docker/app/Dockerfile

```
@@ -18,6 +18,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
       python3-dev \
     && rm -rf /var/lib/apt/lists/*
 
+# Install Node.js 20.x LTS
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)